docs/ops/shape/Unsqueeze_1.md

```diff
@@ -10,23 +10,23 @@
 
```

Comment:
nit: missing *Detailed Description* section, not sure if applicable here, but wanted to confirm that really we can't say much more than in short description

Reply:
It's a simple op, I decided that a detailed description was not needed in this case.